File: URI/pulasapo.py
infos = [int(x) for x in input().split()]
salto = infos[0]
qte = infos[1]
win = True

canos = [int(x) for x in input().split()]

for i in range(qte-1):
    aux = canos[i] - canos[i+1]
    if  aux < -salto or aux > salto:
        
        win = False
        break

if win == True:
    print("YOU WIN")
else:
    print("GAME OVER")

    

File: URI/pulasapo.lua
salto = io.read("*number")
qte = io.read("*number")
canos = {}
win = true
for i=1,qte
do
    canos[i] = io.read("*number")   
end
for i=1,qte-1
do
    aux = canos[i]-canos[i+1]
    if aux < -salto or aux > salto then
        win = false
        break
    end
end
if win == true then
    print("YOU WIN")
else
    print("GAME OVER")
end